### PR TITLE
Updating tools/canu from version 2.1.1 to 2.2

### PR DESCRIPTION
**Pull request by planemo-autoupdate**
Hello! This is an automated update of the following tool: **tools/canu**. I created this PR because I think the tool's main dependency is out of date, i.e. there is a newer version available through conda.

I have updated tools/canu from version 2.1.1 to 2.2.

**Project home page:** https://github.com/marbl/canu/releases

### CODE REVIEW

**Comment by bgruening** [comment]
Yes, I think so. Assembly do change and are hard to test.

File: tools/canu/canu.xml [hunk 1]
```diff
@@ -204,16 +204,18 @@
...
                 </assert_contents>
             </output>
         </test>
         <!--trimming test - it does currently not trim anything due to the input data -->
         <test expect_num_outputs="2">
             <param name="inputs" ftype="fasta" value="ecoli-reads.fasta"/>
             <param name="technology" value="-nanopore"/>
+            <param name="processing" value="-corrected"/>
+            <!-- <param name="minInputCoverage" value="1"/> -->
             <param name="genomeSize" value="3.4m"/>
```
Comment (gallardoalba) [inline tools/canu/canu.xml:219]:
```suggestion
            <param name="genomeSize" value="0.01m"/>
```

File: tools/canu/canu.xml [hunk 2]
```diff
@@ -56,20 +56,20 @@
...
+        utgovlMemory=\${GALAXY_MEMORY_MB:-6144}M
         corThreads=\${GALAXY_SLOTS:-4}
-        corMemory=\${GALAXY_MEMORY_MB:-4096}M
+        corMemory=\${GALAXY_MEMORY_MB:-6144}M
         cnsThreads=\${GALAXY_SLOTS:-4}
-        cnsMemory=\${GALAXY_MEMORY_MB:-4096}M
-        oeaMemory=\${GALAXY_MEMORY_MB:-4096}M
+        cnsMemory=\${GALAXY_MEMORY_MB:-6144}M
+        oeaMemory=\${GALAXY_MEMORY_MB:-6144}M
         oeaThreads=\${GALAXY_SLOTS:-4}
```
Comment (gallardoalba) [inline tools/canu/canu.xml:73]:
```suggestion
        oeaThreads=\${GALAXY_SLOTS:-4}
        merylThreads=\${GALAXY_SLOTS:-4}
        merylMemory=\${GALAXY_MEMORY_MB:-6144}M
        corovlThreads=\${GALAXY_SLOTS:-4}
        corovlMemory=\${GALAXY_MEMORY_MB:-6144}M
```

**Review by gallardoalba — COMMENTED** [review]
Now it will pass the tests :)

**Review by simonbray — APPROVED** [review]

**Review by bgruening — APPROVED** [review]

File: tools/canu/canu.xml [hunk 3]
```diff
@@ -56,21 +56,25 @@
...
+        merylThreads=\${GALAXY_SLOTS:-4}
+        merylMemory=\${GALAXY_MEMORY_MB:-6144}M
```
Comment (gallardoalba) [inline tools/canu/canu.xml:75]: I would revert it because I think that Meryl requites to provide the memory in Gigabytes.

**Pull request merged**